frontend/src/components/FormControl/FormLabel/FormLabel.tsx

```diff
@@ -81,6 +87,14 @@ export const FormLabel = ({
       flexDir="column"
       overflowWrap="break-word"
       {...labelProps}
+      sx={{
+        ...(isMRFPrefilled && {
+          _disabled: {
```

Comment:
Could this be applied at the `theme/component/FormLabel.tsx`? This would make the change to be consistent throughout the code. 

1. Updating component to accept new modifiers
2. Updating component theme to change css based on the new modifer